stable/sonarqube/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 0.13.1
+version: 0.13.2
```

Comment:
Please bump the minor version instead as this adds a feature

Reply:
I think it is a bug fix because we already have a feature to allow updating the sonar.properties for sonarqube (which also includes the sonarqube context path in it) using sonarProperties variable. but when someone updates that specific property, the deployment fails.
Suggestions ?

Reply:
Sounds good